msgid ""
msgstr ""
"Project-Id-Version: pylav\n"
"POT-Creation-Date: 2022-12-30 22:38+0000\n"
"PO-Revision-Date: 2023-01-21 17:14\n"
"Last-Translator: \n"
"Language-Team: Swedish\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: redgettext 3.4.2\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Crowdin-Project: pylav\n"
"X-Crowdin-Project-ID: 535244\n"
"X-Crowdin-Language: sv-SE\n"
"X-Crowdin-File: /pylav/extension/bundled_node/locales/messages.pot\n"
"X-Crowdin-File-ID: 721\n"
"Language: sv_SE\n"

#: pylav/extension/bundled_node/manager.py:227
msgid "You are attempting to run the managed Lavalink node on an unsupported machine architecture"
msgstr "Du försöker köra den hanterade Lavalink-noden på en maskinarkitektur som inte stöds"

#: pylav/extension/bundled_node/manager.py:452
msgid "The output of `{java_exc} -version` was unexpected\n"
"{version_info}"
msgstr "Utmatningen av `{java_exc} -version` var oväntad\n"
"{version_info}"

#: pylav/extension/bundled_node/manager.py:472
msgid "Port {port} already in use. Managed Lavalink startup aborted"
msgstr "Port {port} används redan. Hanterad Lavalink start avbröts"

#: pylav/extension/bundled_node/manager.py:477
msgid "Lavalink failed to start: {line}"
msgstr "Lavalink kunde inte starta: {line}"

